openapi-core/src/main/java/io/ballerina/openapi/core/generators/schema/ballerinatypegenerators/AllOfRecordTypeGenerator.java

```diff
@@ -98,15 +122,22 @@ private List<Node> generateAllOfRecordFields(List<Schema> allOfSchemas) throws B
         List<Node> recordFieldList = new ArrayList<>();
         for (Schema allOfSchema : allOfSchemas) {
             if (allOfSchema.get$ref() != null) {
+                String extractSchemaName = GeneratorUtils.extractReferenceType(allOfSchema.get$ref());
```

Comment:
To make the variable name a noun. 
```suggestion
                String extractedSchemaName = GeneratorUtils.extractReferenceType(allOfSchema.get$ref());
```